consensus/dummy/consensus.go

```diff
@@ -384,13 +362,16 @@ func (eng *DummyEngine) FinalizeAndAssemble(chain consensus.ChainHeaderReader, h
 	)
 	if config.IsSubnetEVM(header.Time) {
 		// Verify that this block covers the block fee.
-		if err := eng.verifyBlockFee(
-			header.BaseFee,
-			headerExtra.BlockGasCost,
-			txs,
-			receipts,
-		); err != nil {
-			return nil, err
+		if !eng.consensusMode.ModeSkipBlockFee {
+			if err := customheader.VerifyBlockFee(
+				header.BaseFee,
+				headerExtra.BlockGasCost,
+				txs,
+				receipts,
+				nil,
```

Comment:
Above in `Finalize` it looks like we declare `contribution`, but that it is always left as `nil`. Here, we just pass `nil` directly. I assume the benefit of declaring the unassigned var is to reduce the diff with `coreth`, but think we should be consistent with either declaring or not (I'd personally lean not declaring just to reduce the diff, because it's not clear in the code itself why the var is there if it's never assigned).

Reply:
Yeah I think this is generally a more philosophical question. I think I'd be inclined to agree if the merger wasn't something that hopefully is happening in the near term. Muddying the subnet-evm code is absolutely valid, but it but it hopefully won't be muddied for long (as it will cease to exist), and I'd further hope that it wouldn't matter that much for people looking at it, as I'd hope no one would be looking at it - are we going to be making consensus specific subnet-evm changes prior to the repository merger? Surely the answer is no. 

@ceyonur do you have thoughts?